docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -569,6 +570,15 @@ To configure this setting globally, set `proxy-buffer-size` in [NGINX ConfigMap]
 nginx.ingress.kubernetes.io/proxy-buffer-size: "8k"
 ```
 
+### Proxy HTTP version
+
+Using this annotation sets the [`proxy_http_version`](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_http_version) that the Nginx reverse proxy will use to communicate with the backend.
+By default this is set to "1.1".  Note this is *not* the same as the stock Nginx default which is "1.0".
```

Comment:
Please remove `Note this is *not* the same as the stock Nginx default which is "1.0".`